jax/experimental/jax2tf/jax2tf.py

```diff
@@ -904,13 +908,66 @@ def _reduce_window(jax_f, reducer, init_val, operand, window_dimensions,
                             window_dilations=window_dilation, padding=padding)
   out.set_shape(out_shape)
   return out
+
+def _get_max_identity(dtype):
+  jax_dtype = to_jax_dtype(dtype)
+  val = None
+  if dtypes.issubdtype(jax_dtype, np.inexact):
+    val = -np.inf
+  elif dtypes.issubdtype(jax_dtype, np.integer):
+    val = dtypes.iinfo(jax_dtype).min
+  elif dtypes.issubdtype(jax_dtype, np.bool_):
+    val = False
+  return _safe_convert_to_tensor(val, dtype)
```

Comment:
Maybe change the code to make it obvious that val cannot be None.

Reply:
Addressed in `eee7daa`